stackslib/src/chainstate/stacks/boot/mod.rs

```diff
@@ -1798,7 +1794,7 @@ pub mod test {
             boot_code_test_addr(),
             POX_4_NAME,
             "stack-extend",
-            vec![Value::UInt(lock_period), addr_tuple],
+            vec![Value::UInt(lock_period), addr_tuple, Value::Optional(OptionalData {data: None})],
```

Comment:
Don't pass `None`; instead, make this an argument in Rust which gets passed to `stack-extend`.